debian/changelog

```diff
@@ -0,0 +1,5 @@
+laarid-meta (20161210) testing; urgency=low
```

Comment:
I think this should be [Android API Level](https://developer.android.com/guide/topics/manifest/uses-sdk-element.html#ApiLevels).